addons/go.mod

```diff
@@ -68,6 +68,7 @@ require (
 	github.com/apparentlymart/go-cidr v1.1.0 // indirect
 	github.com/asaskevich/govalidator v0.0.0-20210307081110-f21760c49a8d // indirect
 	github.com/aunum/log v0.0.0-20200821225356-38d2e2c8b489 // indirect
+	github.com/avinetworks/sdk v0.0.0-20201123134013-c157ef55b6f7 // indirect
```

Comment:
question: are these go mod changes under `addon package` expected ?

Reply:
It looks like this dependency is used by other modules which add imported, I run `go mod why -m github.com/avinetworks/sdk` and got the following output:
```
# github.com/avinetworks/sdk
github.com/vmware-tanzu/tanzu-framework/addons/pkg/util
github.com/vmware-tanzu/tanzu-framework/addons/pkg/util.test
github.com/vmware-tanzu/tanzu-framework/pkg/v1/tkg/fakes
github.com/vmware-tanzu/tanzu-framework/pkg/v1/tkg/client
github.com/vmware-tanzu/tanzu-framework/pkg/v1/tkg/avi
github.com/avinetworks/sdk/go/clients
```